packages/siopv2-oid4vp-rp-rest-client/__tests__/restAgent.test.ts

```diff
@@ -0,0 +1,37 @@
+import { createAgent, IResolver } from '@veramo/core'
+import { SiopV2OID4VpRpRestClient } from '../src'
+
+const definitionId = '9449e2db-791f-407c-b086-c21cc677d2e0'
+const baseUrl = 'https://ssi-backend.sphereon.com'
+
+const agent = createAgent<IResolver>({
+  plugins: [new SiopV2OID4VpRpRestClient(baseUrl, definitionId)],
+})
+
+describe('@sphereon/siopv2-oid4vp-rp-rest-client', () => {
+  it('should call the endpoint for deleteDefinitionCorrelation', async () => {
+    const authRequest = await agent.generateAuthRequest({})
+    agent.removeAuthRequestSession({
+      correlationId: authRequest.correlationId,
+    })
+    await new Promise((f) => setTimeout(f, 5000))
```

Comment:
Why wait 5 seconds?. Rather would see a shorter timeout, or a while loop with guard until it is either deleted or 5 seconds have expired

Reply:
just wanted to be safe on this. in the new version (in the ssi-sdk) you're returning a response, so I will not need this anymore once that changed, I'll just `await` here